### PR TITLE
Fix syntax errors in .travis.yml

### PR DESCRIPTION
**Pull request by jsantos17**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,10 +38,10 @@ jobs:
   include:
     - stage: test
       script:
-      - set -e
-      # Just protecting against automated scrapers
-      - base64 -d testCredentials.json.b64 > testCredentials.json
-      - $SBT ++$TRAVIS_SCALA_VERSION test
+        - set -e
```
Comment (jsantos17) [inline .travis.yml]: This isn't bad YAML, but reindented it to match the block below.

File: .travis.yml [hunk 2]
```diff
@@ -53,8 +53,8 @@ jobs:
         - export -f bumpInDownstream
         - scripts/bumpDependentProject slamdata slamx bumpInDownstream
 
-    git:
-      depth: false
+    - git:
+        depth: false
```
[the remark this answers is not in the document]
Comment (jsantos17) [reply .travis.yml]: That works and it makes more sense too. I have found documentation on `git: depth: false`, but I'm not sure where is that setting allowed. `travis lint` doesn't help much.

[the remark this answers is not in the document]
Comment (jsantos17) [reply .travis.yml]: Tried to make `git` top-level, but `travis lint` fails then, with this puzzling error:

```
server error (500: "Sorry, we experienced an error.\n\nrequest_id:d045fbd9-c374-42a4-940e-f3cbe6fab43d\n")
```

[the remark this answers is not in the document]
Comment (nicflores) [reply .travis.yml]: @jsantos17, I moved it over to be at the same level and `script` and the linter seems to be ok:
```
      script:
        ....
      git:
        depth: false
```

[the remark this answers is not in the document]
Comment (jsantos17) [reply .travis.yml]: I guess the only way to know if this actually works is to see Travis logs.

[the remark this answers is not in the document]
Comment (nicflores) [reply .travis.yml]: yeah, lets see what happens when "real travis" takes a crack at reading the `.travis.yml` file.

[the remark this answers is not in the document]
Comment (jsantos17) [reply .travis.yml]: Anyway, anything that is valid yaml is an improvent in this case :)

**Pull request merged**